jobs/jobs/db_service.py

```diff
@@ -224,3 +224,13 @@ def change_job(
         if new_job_params_dict[job_param_key] is not None:
             setattr(job, job_param_key, new_job_params_dict[job_param_key])
     db.commit()
+
+
+def is_job_exists_by_name(db: Session, job_name: str) -> bool:
```

Comment:
let's make this function more generic - get job by name. You could utilize that in numerous places later. No need to check for existence here